spikingjelly/activation_based/base.py

```
@@ -9,13 +9,13 @@
 try:
     import cupy
 except BaseException as e:
-    logging.info(f"spikingjelly.activation_based.base: {e}")
+    logger.debug("spikingjelly.activation_based.base: %s", e)
     cupy = None
 
 try:
     import triton
 except BaseException as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `BaseException`

(BLE001)